homeassistant/components/zwave/__init__.py

```diff
@@ -627,6 +628,39 @@ def value_changed(self, value):
         self.update_properties()
         self.schedule_update_ha_state()
 
+    def value_handler(self, method='get', class_id=None, index=None,
```

Comment:
That was my initial thought, but since the code is nearly the same for both, I found it better to have an argument.